### PR TITLE
Add Restaumatic

### PR DESCRIPTION
**Pull request by kozak**
Hi @ajnsit would you mind adding us please? :) Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Some of these companies support remote hires. Where that information is availabl
 [Middlebury Interactive Languages](http://www.middleburyinteractive.com/) | Middlebury, Vermont, USA | Education/Saas | [Reddit](https://www.reddit.com/r/purescript/comments/3mzotb/using_purescript_for_work/cvkerae/) | ?
 [Monadfix](https://monadfix.io) | Multiple locations | Consulting/Training | [Website](https://monadfix.io) | Yes
 [Oxford Abstracts](https://oxfordabstracts.com) | London, UK. Kilchoan, UK. Barcelona, Spain. Budapest, Hungary | Academic conferences | [Open source project](https://github.com/OxfordAbstracts/purescript-graphql-client) | Yes
+[Restaumatic]([https://slamdata.com](https://www.restaumatic.com/en/) | Warsaw, Poland | Restaurant e-commerce | [GitHub](https://github.com/restaumatic?language=purescript) | Yes
```
Comment (ajnsit) [inline README.md]: Hi, the slamdata link seems to be in error?

Comment (ajnsit) [reply README.md]: Ping @kozak

Comment (kozak) [reply README.md]: Totally missed this. Thank you for your vigilance! Fixing!

Comment (kozak) [reply README.md]: Done

**Comment by ajnsit** [comment]
Thanks!

**Comment by kozak** [comment]
Thank you :)

**Pull request merged**